app/views/podcast_engagement/show.html.erb

```diff
@@ -1,7 +1,47 @@
+<% content_for :title, @podcast.title.to_s %>
+
 <%= render "podcasts/tabs" %>
 
-<h1>Editing podcast engagement</h1>
+<%= form_with(model: @podcast, url: podcast_engagement_path, html: {autocomplete: "off"}, data: {controller: "unsaved", unsaved_changed_value: @podcast.changed?, unsaved_confirm_value: t(".confirm")}) do |form| %>
+<div class="row mt-4">
+    <div class="col-8">
+        <div class="form-floating input-group">
+            <%= form.text_field :donation_url %>
+            <label for="floatingInput"><%= t(".donation_link") %></label>
+        </div>
+
+            <hr style="height:.1rem background-color:black">
+
+            <h3 aria-label="Micropayments"><%= t(".micropayments") %></h3>
+
+            <div class="form-floating input-group">
+                <%= form.text_field :payment_pointer %>
+                <label for="floatingInput"><%= t(".payment_pointer") %></label>
+                <span class="material-icons mx-auto justify-content-center col-1 border px-auto">link</span>
+            </div>
+
+        <div>
+            <hr style="height:.1rem background-color:black">
+            <h3 aria-label="Subscribe and Follow Links">Subscribe and Follow Links</h3>
+            <h3 aria-label="Subscribe and Follow Links">Social Media Links</h3>
+        </div>
+    </div>
+
+    <div class="col-lg-4">
+    <section class="card">
+        <div class="card-header-info">
+            <h5 class="card-title fw-bold">Status</h5>
+        </div>
 
-<%= render "form", podcast_engagement: @podcast %>
+        <div class="card-footer text-start">
+            <strong><%= t(".last_saved") %></strong>
+            <datetime><%= l(@podcast.updated_at) %></datetime>
+            <div class="btn-group">
+                <%= form.submit class: "btn btn-primary" %>
```

Comment:
Will adjust so the button and `Last Saved` text are side by side as in the [figma doc.](https://www.figma.com/proto/Yewv83Q8btHcaOBWCtGgKs/PRX-Dovetail-with-Navigation?page-id=80%3A2947&node-id=2450-25654&viewport=798%2C-3032%2C0.25&scaling=min-zoom&starting-point-node-id=80%3A3420&hide-ui=1)

Reply:
This seems pretty similar to the `views/podcasts/_form_status.html.erb`?  Which I fixed to be side-by-side in #629, if you want to dig through that code for an example.